doc/tutorials/gpu-passthru.rst

```diff
@@ -0,0 +1,147 @@
+.. _gpu-passthrough:
+
+Enable GVT-d in ACRN
+#########################################
+
+This tutorial describes how to enable GVT-d in ACRN.
```

Comment:
If I understand this tutorial correctly, we are starting from a Service VM that's been set-up correctly already and we will launch a Windows User VM (WaaG) that will take over the GPU using GVT-d, is that correct?

I think it would be beneficial to mention this more explicitly in this intro.

Does it also mean that the user should really be using an SSH (or serial port) session when doing all this? I'm assuming so since the Windows User VM will be given the GPU when launched.

Reply:
What tutorial/getting started guide should be followed before starting here?

If I look at the https://projectacrn.github.io/latest/tutorials/using_windows_as_uos.html for example, it uses the SDC scenario by default. Is that appropriate?

Reply:
I don't think we need to mention this **Service VM that's been set-up correctly already and we will launch a Windows User VM (WaaG) that will take over the GPU using GVT-d**. This tutorial shows the usage the `acrn-dm`, only Servicee VM can do this, So we don't need to mention.
The user can choose SSH session or not, it depends on their choice.

Reply:
GVT-d is not limited to SDC  scenario. It's used when needing HMI User VM.

Reply:
Yes. It's better to add Prerequisites. I will add it.

Reply:
> The user can choose SSH session or not, it depends on their choice.

What are the options left to interact with the Service VM once the User VM with GVT-d has been launched? The reason why I brought this up is that many users have the Service VM up and running on the monitor (GNOME desktop) and use the local keyboard/mouse. Does the User VM take over that monitor once it's been launched?

Reply:
> > The user can choose SSH session or not, it depends on their choice.
> 
> What are the options left to interact with the Service VM once the User VM with GVT-d has been launched? The reason why I brought this up is that many users have the Service VM up and running on the monitor (GNOME desktop) and use the local keyboard/mouse. Does the User VM take over that monitor once it's been launched?

yes. The User VM take over that monitor once it's been launched

Reply:
> yes. The User VM take over that monitor once it's been launched

This will catch a few people by surprise. Can you add a note that it is recommended to have either a serial port or SSH session open in the Service VM to be able to continue interact with it?

Reply:
> > yes. The User VM take over that monitor once it's been launched
> 
> This will catch a few people by surprise. Can you add a note that it is recommended to have either a serial port or SSH session open in the Service VM to be able to continue interact with it?

ok